client/settings/general-settings-section/payment-methods-list.js

```diff
@@ -92,17 +83,7 @@ const GeneralSettingsSection = () => {
 
 				return (
 					<ListElement key={ method }>
-						{ isUpeEnabled && (
-							<CheckboxControl
```

Comment:
Moved this to its own file in `client/settings/general-settings-section/payment-method-checkbox.js` - it will be useful once we have to encapsulate this: 
![Screen Shot 2021-09-29 at 6 41 16 PM](https://user-images.githubusercontent.com/273592/135363064-20b552ca-8a57-46f4-a478-d3c6edb87d6b.png)